File: PrintPrime.java
import java.util.*;
public class PrintPrime {
public static void main(String[]args){
    Scanner input = new Scanner(System.in);
    System.out.print("Enter low range num and high range num separeted by white space: ");
    
    int low = input.nextInt();
    int high = input.nextInt();

    for(int n = low; n <= high; n++){
        
        int count = 0;
        for(int div = 2; div*div <= n; div++){
            if(n % div == 0){
                count++;
                break;
            }
        }
        if(count == 0){
            System.out.println(n);
        }
        
    }
}
}   

File: PrintPrime.c
#include <stdio.h>
void main(){
	int low,high,n,count,div;
	printf("Enter low range num and high range num separeted by white space: ");
	scanf("%d %d",&low,&high);
	
	
	for (n = low; n <= high; n++){

		count = 0;
		for(div = 2; div * div <= n; div++){
			if(n % div == 0){
				count++;
				break;
			}
		}
		if(count==0){
			printf("%d\n",n);//print all prime number between range
		}
	}
}
